src/containers/Settings/components/FormToggle.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './FormToggle.module.scss';
+import { Text, Toggle } from 'components';
+import { LinkedAvatar } from 'explorer-components';
+import { FormInput } from 'hocs';
+
+const FormToggle = props => {
```

Comment:
hmmm - this likely should not be generically FormToggle but rather something more specific.
Also, Why is this wrapped in a container-fluid??

Reply:
I don't think this needs flexbox grid either - lets talk about this.